TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/210_Motion_Vectors/MotionVecDebug.shader

```diff
@@ -0,0 +1,65 @@
+Shader "MotionVecDebug"
```

Comment:
Is this used for testing only? Maybe it should be under the `https://github.com/Unity-Technologies/Graphics/tree/master/com.unity.testing.urp` package.

Reply:
Minor naming suggestion
```suggestion
Shader "MotionVectorsDebug"
```

Reply:
There doesn't seem to be any other test related shaders in that package, why should this one specifically be moved? Other tests have shaders kept in the foundation project.